app/src/main/res/values/strings.xml

```diff
@@ -263,4 +267,10 @@
     <string name="expand_button_title" tools:override="true">Expand</string>
     <string name="undo">Undo</string>
     <string name="per_game_settings_active_message">Per-game settings are active</string>
+    <string name="delete_save_confirmation_message">Are you sure you want to delete this save?</string>
+    <string name="action_irreversible">This action is irreversible</string>
+    <string name="yes">Yes</string>
+    <string name="no">No</string>
+    <string name="save_file_imported_ok">The save file was imported successfully</string>
+    <string name="save_file_invalid_zip_structure">Failed to unzip. First subfolder must be named after the TitleId of the game.</string>
```

Comment:
`Invalid Zip directory structure: the first subfolder name must be the Title ID of the game.`